packages/typespec-ts/src/modular/emitSamples.ts

```diff
@@ -379,7 +379,7 @@ function prepareExampleParameters(
   const bodyParam = method.operation.bodyParam;
   const bodySerializeName = bodyParam?.serializedName;
   const bodyExample = parameterMap[bodySerializeName ?? ""];
-  if (bodySerializeName && bodyExample && bodyExample.value) {
```

Comment:
we need to add a UT: https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/test/modularUnit/scenarios/samples/parameters/paramaterSpreadCheck.md#L37.